src/features/homepage/css/Card.css

```diff
@@ -0,0 +1,38 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  text-align: right;
+  padding: 7px;
+
+  /* height: 130px; */
+}
+
```

Comment:
- [ ] Let's try not to have commented-on code in our codebase as they make our code dirty and not look professional. It's a good convention to not leave unused codes commented in our codebase. Please remove all commented codes from your codebase 👍🏽